payments-core/src/main/java/com/stripe/android/payments/paymentlauncher/PaymentLauncherFactory.kt

```diff
@@ -46,21 +46,28 @@ class PaymentLauncherFactory(
     fun create(
         publishableKey: String,
         stripeAccountId: String? = null
-    ): PaymentLauncher = StripePaymentLauncher(
-        { publishableKey },
-        { stripeAccountId },
-        hostActivityLauncher,
-        context,
-        BuildConfig.DEBUG,
-        Dispatchers.IO,
-        Dispatchers.Main,
-        StripeApiRepository(
+    ): PaymentLauncher {
```

Comment:
If we're using this on Payment Sheet we'll need a constructor where all of this can be injected, right?

Reply:
The same constructor is used in both public API and `PaymentSheet`.
For public usage, we call it manually here, for `PaymentSheet` we [inject](https://github.com/stripe/stripe-android/blob/0def5760640ace6addaa44d19e4e783fb9f04d46/paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheetViewModel.kt#L78) through [StripePaymentLauncherAssistedFactory](https://github.com/stripe/stripe-android/blob/a1a2133625880241f92bf533919c46d188f461e9/payments-core/src/main/java/com/stripe/android/payments/paymentlauncher/StripePaymentLauncherAssistedFactory.kt#L19)